### PR TITLE
Changelog-redesign-image-component

### PR DESCRIPTION
**Pull request by lindseybradford**
Ticket: https://www.notion.so/mxpnl/Update-formatting-of-photos-on-individual-post-pages-11ce0ba925628088a232c3e66f580e42?pvs=4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,8 +41,11 @@
     "@types/node": "22.5.2",
     "autoprefixer": "^10.4.20",
     "cspell": "^8.14.2",
+    "install": "^0.13.0",
+    "npm": "^10.9.0",
```
Comment (ryancurtis1) [inline package.json]: seems like these two packages were mistakenly added?

File: components/ImageFrame/ImageFrame.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+// https://www.tailwind-variants.org/docs
+import { tv } from "tailwind-variants";
+
+type ImageFrameProps = {
+  src: string;
+  alt?: string;
+};
+
+export default function ImageFrame({
+  alt = "Thumnail of screenshot",
```
Comment (junhouse) [inline components/ImageFrame/ImageFrame.tsx]: Spelling

File: components/ImageFrame/ImageFrame.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  const [width, setWidth] = useState(0);
+
+  useLayoutEffect(() => {
+    if (imageRef.current) {
+      setHeight(imageRef.current.getBoundingClientRect().height);
+      setWidth(imageRef.current.getBoundingClientRect().width);
+    }
+  }, []);
+
+  const isTall = height > 400;
```
Comment (junhouse) [inline components/ImageFrame/ImageFrame.tsx]: can we give a name for `400`?

File: package.json (hunk 1 above)
Comment (junhouse) [reply package.json]: Could we remove if not used?

File: components/changelog-index.js [hunk 4]
```diff
@@ -1,20 +1,14 @@
...
-import ExtendedButton from "/components/ExtendedButton/ExtendedButton";
+import { ImageFrame } from "/components/ImageFrame";
 import Link from "next/link";
 
 const renderMedia = (page) => {
   if (page.frontMatter?.thumbnail) {
     return (
-      <img
+      <ImageFrame
         src={page.frontMatter.thumbnail}
```
Comment (junhouse) [inline components/changelog-index.js]: looks like this conflicts w/ https://github.com/mixpanel/docs/pull/1553/files ?

File: package.json (hunk 1 above)
Comment (ryancurtis1) [reply package.json]: 👍

**Pull request merged**